convert/src/label_studio_to_badgerdoc/badgerdoc_format/pdf_converter.py

```diff
@@ -0,0 +1,63 @@
+from typing import List
+
+from pdfminer.high_level import extract_pages
+from pdfminer.layout import LTChar, LTTextBoxHorizontal, LTTextLineHorizontal
+
+from src.label_studio_to_badgerdoc.models import (
+    BadgerdocToken,
+    Offset,
+    Page,
+    PageSize,
+)
+
+
+class PlainPDFToBadgerdocTokensConverter:
+    def __init__(self):
+        self.offset = 0
+        self.page_size = None
+
+    def get_bbox(self, char: LTChar) -> tuple:
+        return (
+            char.x0,
+            self.page_size.height - char.y1,
+            char.x1,
+            self.page_size.height - char.y0,
+        )
+
+    def convert_line(self, line: LTTextLineHorizontal) -> List[BadgerdocToken]:
+        tokens = []
+        for char in line:
+            if not isinstance(char, LTChar):
+                continue
+            tokens.append(
+                BadgerdocToken(
+                    bbox=self.get_bbox(char),
+                    text=char.get_text(),
+                    offset=Offset(begin=self.offset, end=self.offset + 1),
+                )
+            )
+            self.offset += 1
```

Comment:
I believe it is better this way:

if isinstance(char, LTChar):
    logic...

"else" is not needed

Reply:
same here was changed after review